about.html

```diff
@@ -0,0 +1,137 @@
+              <hr />
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section id="about-hero">
+      <div class="flex-container">
+        <h2>"Hello Aspiring Thrillers!"</h2>
+        <h1>About Air Sports Club</h1>
+        <div class="about-hero__message">
+          <p>
+            ​At Air Sports Club, we specialize in small group for multiple air
+            sport adventures. Over the past 28 years, Air Sports Club has grown
+            rapidly to become one of the leading sport club like adventure
+            companies. We Provide Instructors to help you do the air actively
+            and explore the adventure of Skydiving, Paragliding,
+            Paramotoring,Hang Gliding, Wingsuit Flying and more. We always dig
+            deeper in our new destinations to keep evolving our new locations or
+            spot with the core intention of keeping your experiences truly
+            original and the memories priceless. When you are doing this
+            activities, you just feel more Thrilled. When you see more
+            activities, you meet more people, create more lasting memories...
+            When you trust in our expertise to deliver you these extraordinary
+            experiences, it truly means the world to us.
+          </p>
+        </div>
+        <h4 class="about-hero__contact"></h4>
+          Please contact us on the email below for any further queries <br />
+          about our services.
+        </h4>
+        <a class="about-hero__email" href="#Contact"
+          >ashwincarvalho27@gmail.com</a
+        >
+      </div>
+    </section>
+    <section id="about-logo">
+      <div class="flex-container">
+        <h2>Air Sports Club Logo</h2>
+        <hr class="section__hr" />
+        <p>
+          The Air Sports Club logo was decided after a logo design competition
+          from 5th of June to 7th of July during the company logo contest, even
+          the friends of the Instructors took part and had great fun in the logo
+          contest.
+        </p>
+        <div class="about-logo__show">
+          <img src="assets/images/air-sports-logo.jpg" alt="Air Sports Club" />
+        </div>
+      </div>
+    </section>
+    <section id="about-past">
+      <div class="flex-container">
+        <h2>Recent Sports Events</h2>
+        <hr class="section__hr" />
+        <p>
+          Take a look at the last two Sports Events which took place in Jan 2021
+          and Feb 2021
+        </p>
+        <div class="about-past__grid">
+          <article class="about-past__cards card1"></article>
+          <article class="about-past__cards card2"></article>
+        </div>
+      </div>
+    </section>
+
+    <section id="footer">
+      <div class="flex-container">
+        <img src="assets/images/air-sports-logo.jpg" alt="Air Sports Logo" />
+        <p>Copyright ©️ 2022 Air Sports Club. Some Rights Reserved</p>
+      </div>
+    </section>
```

Comment:
- [ ] For best practices of HTML as the previous code reviewers required, kindly use the `footer` tag instead of the `section` tag. 👆🏽

Reply:
Solved.